portfolio/src/main/webapp/script.js

```diff
@@ -12,19 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
-
 /**
  * Adds a list of hobbies to the page by displaying the hobby's description text
  * and updating the wrapper's background color so that it appears 
  */
 function addHobby() {
-  var hobbies = 'playing the Clarinet, reading mystery novels, (trying to) speak Spanish';
+  let hobbies = 'playing the Clarinet, reading mystery novels, (trying to) speak Spanish';
```

Comment:
Can you use `const` instead of `let` here? I don't think you need to mutate this at all.

Reply:
Done